web/src/cypress/support/page-action.ts

```diff
@@ -1,8 +1,19 @@
-export function pageAction(s: string): Cypress.Chainable {
+declare global {
+  namespace Cypress {
+    interface Chainable {
+      /** Click an action from the page-level "Other Actions" menu. */
+      pageAction: typeof pageAction
+    }
+  }
+}
+
+function pageAction(s: string): Cypress.Chainable {
   return cy
     .get('[data-cy=app-bar]')
     .find('button[data-cy=other-actions]')
     .menu(s)
 }
 
 Cypress.Commands.add('pageAction', pageAction)
+
+export {}
```

Comment:
What are these empty exports used for?

Reply:
To force typescript into module mode so that they can all be written the same way, in the same scope:
https://www.typescriptlang.org/docs/handbook/2/modules.html#non-modules

Reply:
Cool, and I imagine it's fine not exporting the functions themselves now since they're added onto `cy` anyways. Thanks!